_example/work.go

```diff
@@ -1,24 +1,44 @@
 package example
 
 import (
-	"math/rand"
-	"strconv"
+	"fmt"
 	"time"
 )
 
 // Work loops forever, generating a bunch of allocations of various sizes in
 // order to force the garbage collector to work.
 func Work() {
-	m := map[string][]byte{}
+	m := make(map[int]interface{})
+	i := 0
+	for ; ; i++ {
```

Comment:
Perhaps this could be written as

for i:= 0; ; i++ {

Reply:
Indeed